Wrapping/Generators/Python/Tests/verifyGetOutputAPIConsistency.py

```diff
@@ -21,6 +21,11 @@
 import itk
 import sys
 from itk.support.template_class import itkTemplate
+from itk.support.base import load_factories
+load_factories('ImageIO')
+load_factories('MeshIO')
+load_factories('TransformIO')
+load_factories('FFTImageFilter')
```

Comment:
Init

Reply:
I think `force_load` was not loading the factories. It may be fine if it does as a side-effect, but I do not think it necessarily should.

Reply:
Not sure I understand. What is the purpose of `force_load` if not to force all lazy modules to be loaded?